### DISCUSSION TITLE
Calendar positioned on the Ecliptic line

### DISCUSSION THREAD

**Discussion by PLANETARAS** [Stellarium/stellarium#2904] [Ideas]
In the folder Skycultures - Sardinian I created textures of the 12 months of the year considered files.png as fictitious constellations positioned on the Ecliptic. I performed the placement by altering a file constellationship.fab and constellationart.fab adding 12 records named (for example: 001 January.png 13 1180 104365 1418 57 96327 2970 1213 88839. The numbers after the file indicate a star and its coordinates in pixels in textures). The result is the possibility to select Skyculture Sardinian and see superimposed on the Ecliptic the calendar with the Sun correctly positioned on the display date and show the trend of its apparent motion during the year.

**Comment by PLANETARAS** [comment]
![200049e3-8955-448b-aa33-3564335d22ea](https://user-images.githubusercontent.com/97767104/206855532-9e449d14-bbca-4306-bc74-4143347b8180.jpg)

**Comment by gzotti** [comment]
And why do you abuse Sardinian skyculture for this?

**Reply by PLANETARAS** [reply]
Because at the moment I do not use the Sardinian culture and then it can be restored at any time because all files are protected in another folder.
